### DISCUSSION TITLE
Multiple breakingpoint within Layout

### DISCUSSION THREAD

**Discussion by cherimo** [quasarframework/quasar#15313] [General - Components / Directives / etc]
Is it possible within Quasar layout to apply different breaking points? I would like to apply the following properties for the next drawer for example.

Mobile view:
```
Breaking point: 500px or less
mini-to-overlay=true
:mini="null"
```

Tablet view
```
Breaking point: Within 500 and 1024px
mini-to-overlay=false
:mini="miniState"
```

Desktop view
```
Breaking point: 1024px or higher
mini-to-overlay=false
:mini="null"
```

I Have looked at several documentations but haven't been able to find any clear information that can answer my question.
How could I realize this within Quasar?

Thank you in advance.

**Comment by metalsadman** [comment]
you can make use of this helper https://quasar.dev/options/screen-plugin to check on screen sizes and apply those props based on it conditionally. ie. `:mini-to-overlay="$q.screen.size.width >= 1024"`.
